src/index.html

```diff
@@ -11,8 +14,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
You can't import '.scss' files into HTML. It should be a '.css' file. You need to compile your '.scss' files into '.css' and then include the '.css' file in your HTML.